### DISCUSSION TITLE
Support for HAP/Homekit

### DISCUSSION THREAD

**Discussion by olierxleben** [microsoft/devicescript#694] [General]
Hi everyone, 

I am new still new to the microcontroller world, but I ask myself if it is within reach to have support for HomeKit / Home Accessory Protocol within device script itself. Or maybe you already support it, but I did not found any information about. 

What do you think is missing to get support?

Greets and thanks for reading, 

Oli

**Comment by pelikhan** [comment]
We did some experiments with HomeBridge --> https://microsoft.github.io/devicescript/samples/homebridge-humidity
